dedupe/convenience.py

```diff
@@ -24,55 +24,88 @@ def consoleLabel(deduper): # pragma: no cover
     '''
 
     finished = False
+    use_previous = False
     fields = unique(field.field
                     for field
                     in deduper.data_model.primary_fields)
 
+    buffer_len = 1 # Max number of previous operations
+    record_label_buffer = []
```

Comment:
this seems like a poor name since it's not just a buffer of labels. What was wrong with `examples`